test/cypress/integration/privacy/privacy.cy.test.js

```diff
@@ -1,14 +1,38 @@
 'use strict'
+const userStubs = require('../../stubs/user-stubs')
+
+const userExternalId = 'a-user-id'
+const gatewayAccountId = 42
+const gatewayAccountExternalId = 'a-valid-account-id'
 
 describe('Privacy page', () => {
-  it('should show the privacy page', () => {
-    cy.visit('/privacy')
+  describe('Logged out user', () => {
+    it('should show the privacy page correctly', () => {
+      cy.visit('/privacy')
+
+      cy.get('h1').should('contain', 'Privacy notice')
+
+      cy.get('[data-cy=sub-navigation]').should('exist')
+      cy.get('[data-cy=sub-navigation] li').should('have.length', 10)
+
+      cy.get('main h2').should('have.length', 10)
+
+      cy.get('[data-cy=breadcrumbs]').should('not.exist')
```

Comment:
Should this test check that the top navigation bar has the "Sign in" link?

Reply:
Good spot - just updated it, please check again.